examples/non-fungible/src/service.rs

```diff
@@ -0,0 +1,236 @@
+// Copyright (c) Zefchain Labs, Inc.
+// SPDX-License-Identifier: Apache-2.0
+
+#![cfg_attr(target_arch = "wasm32", no_main)]
+
+mod state;
+
+use self::state::NonFungibleToken;
+use async_graphql::{EmptySubscription, Object, Request, Response, Schema};
+use async_trait::async_trait;
+use fungible::Account;
+use linera_sdk::{
+    base::{AccountOwner, WithServiceAbi},
+    Service, ServiceRuntime, ViewStateStorage,
+};
+use non_fungible::{NftOutput, Operation, TokenId};
+use std::{
+    collections::{BTreeMap, BTreeSet},
+    sync::Arc,
+};
+use thiserror::Error;
+
+linera_sdk::service!(NonFungibleToken);
+
+impl WithServiceAbi for NonFungibleToken {
+    type Abi = non_fungible::NonFungibleTokenAbi;
+}
+
+#[async_trait]
+impl Service for NonFungibleToken {
+    type Error = Error;
+    type Storage = ViewStateStorage<Self>;
+
+    async fn handle_query(
+        self: Arc<Self>,
+        _runtime: &ServiceRuntime,
+        request: Request,
+    ) -> Result<Response, Self::Error> {
+        let schema = Schema::build(
+            QueryRoot {
+                non_fungible_token: self.clone(),
+            },
+            MutationRoot,
+            EmptySubscription,
+        )
+        .finish();
+        let response = schema.execute(request).await;
+        Ok(response)
+    }
+}
+
+struct QueryRoot {
+    non_fungible_token: Arc<NonFungibleToken>,
+}
+
+#[Object]
+impl QueryRoot {
+    async fn nft(&self, token_id: String) -> Option<NftOutput> {
+        use base64::engine::{general_purpose::STANDARD_NO_PAD, Engine as _};
```

Comment:
This import line seems repeated a few times. Could it be moved to the top of the file?